core/core/initutils.go

```diff
@@ -202,7 +202,10 @@ func getPolicyGetter(ctx context.Context, loadPoliciesFromFile []string, account
 	if accountID != "" && getter.GetKSCloudAPIConnector().GetCloudAPIURL() != "" && frameworkScope {
 		g := getter.GetKSCloudAPIConnector() // download policy from Kubescape Cloud backend
 		return g
+	} else if accountID != "" && getter.GetKSCloudAPIConnector().GetCloudAPIURL() == "" && frameworkScope {
+		logger.L().Ctx(ctx).Warning("Kubescape Cloud API URL is not set, loading policies from cache")
```

Comment:
I am not sure this is warning... wdyt @Daniel-GrunbergerCA  ?

Reply:
Not sure it should be a warning either; maybe a debug print?

Reply:
I mean, if a user is providing his credentials. That essentially means that he is trying to connect to the Cloud API. 
The current behavior doesn't tell the user if he is actually using the API or not

Reply:
I think it makes sense to put this as a `warning`